ui-admin/src/study/participants/enrolleeView/EnrolleeProfile.test.tsx

```diff
@@ -22,6 +22,13 @@ jest.mock('user/UserProvider', () => {
   }
 })
 
+jest.mock('api/api', () => ({
+  ...jest.requireActual('api/api'),
+  fetchEnrolleeAdminTasks: jest.fn(() => Promise.resolve([])),
```

Comment:
there's a shorthand for this that I just learned about...
```suggestion
  fetchEnrolleeAdminTasks: jest.fn().mockResolvedValue([]),
```

Reply:
TIL. I just updated everywhere to use this instead, so Copilot and devs can take advantage